toptal/picasso
# fix: spacing in picasso stories

[FX-4261]

### Description

⚠️ The https://picasso.toptal.net/?path=/story/tutorials-how-to-use-spacings--how-to-use-spacings story is updated in https://github.com/toptal/picasso/pull/3871

This pull request updates spacings in Picasso Storybook. Most of the changes are coming from running the codemod

```
npx @toptal/picasso-codemod v38.1.0 **/*.example.ts*
npx @toptal/picasso-codemod v38.1.0 .storybook/**/*.example.ts*
```

There are custom adjustments (for example, converting `top={7}` to `style={{ paddingTop: '7rem' }}`), they were kept to have the same view of stories.

### How to test


- [Temploy](https://picasso.toptal.net/fx-4261-update-guidelines-on-spacing-usage)
- Check any story that uses `Container` with spacing properties, they should be updated


<img width="1209" alt="Screenshot 2023-09-14 at 22 56 28" src="https://github.com/toptal/picasso/assets/1390758/fdcd4d2c-8abf-499b-9119-6d7723388617">


### Development checks

- [N/A] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [N/A] Annotate all `props` in component with documentation
- [N/A] Create `examples` for component
- [N/A] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [N/A] Covered with tests ([visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md) included)

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[FX-4261]: https://toptal-core.atlassian.net/browse/FX-4261?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
